### ISSUE TITLE
Change Default Org does not show the already connected orgs. Unable to change the Default Org

### ISSUE THREAD

**Issue by 1eandro** [forcedotcom/salesforcedx-vscode#5524]
### Summary

_Short summary of what is going on or to provide context_.
The button 'Change Default Org' is not working.

I'm connected to PROD and UAT. My default org is PROD. I attempt to set it to UAT but the UAT org (or any other org I'm connected to) does not appear.

### Steps To Reproduce:

1. You click in the 'Change Default Org' button, in the bottom status bar, on the left
![image](https://github.com/forcedotcom/salesforcedx-vscode/assets/7105881/9d1edf30-9184-433e-876b-ada8830f1bfe)

1. Then this appears, only the '+' options. The UAT org is not listed there, so you're unable to change it
![image](https://github.com/forcedotcom/salesforcedx-vscode/assets/7105881/9cae12c7-ba7b-4667-bf1d-39c711408463)

### Expected result

It should list the UAT org, or any other org I'm connected to.

Here you can see I'm connected to other orgs
![image](https://github.com/forcedotcom/salesforcedx-vscode/assets/7105881/d352415d-dfcd-49d6-96c8-f207988604fe)

### Actual result

The connected orgs does not appear.

### Additional information

I've already attempted to reconnect, close the vscode, restart the computer, connect again. Nothing of this worked

There's no other vscode extension installed, it's a brand new vscode profile.

**Salesforce Extension Version in VS Code**:
Salesforce Extension Pack (Expanded) - v60.1.2
extension ID: salesforce.salesforcedx-vscode-expanded

**SFDX CLI Version**:
@salesforce/cli/2.32.8 win32-x64 node-v20.11.1

**OS and version**:
Windows 10 Enterprise

**VS Code version**:
Version: 1.80.1
![image](https://github.com/forcedotcom/salesforcedx-vscode/assets/7105881/2be45759-047d-4860-86f0-5260d3e18c79)

**Comment by 1eandro** [comment]
The problem was solved when I've reverted the version of the extension "Salesforce CLI Integration" from "v60.1.2" to "v59.8.0"

**Comment by 1eandro** [comment]
The problem was solved after reverting the version of the extension "Salesforce CLI Integration" from "v60.1.2" to "v59.8.0"

**Comment by AnanyaJha** [comment]
Hi @1eandro , thanks for reporting this issue! Few questions to help us debug: 

- Can you tell us what type of org you're working against? IE scratch org or sandbox? 
- Are you working within a singular SFDX project or switching between projects?
- How did you authorize the UAT org? Was it using the CLI or Extensions? If not the extensions, can you try using that?

Thanks so much!

**Comment by CristiCanizales** [comment]
Hi @1eandro, as Ananya said, you need to update your vscode version. As mentioned in #5390, starting from v59.15.0, our minimum supported version for vscode is 1.82.0.

**Comment by 1eandro** [comment]
Thanks @AnanyaJha and @CristiCanizales, the update of the vscode would take some time due it is for professional use and it takes some time to get approved in our CMDB.

I found the solution by reverting the version a bit, as per my last comment.

I've also created a profile with versions pinned, so the whole ecosystem works as a charm.

**Lesson learned:** _Vscode and SF extensions are constantly updated, so or you keep it up to date, or you pin some specific version where everything works fine_
